src/renate/benchmark/models/base.py

```diff
@@ -87,7 +87,7 @@ def get_extra_state(self, encode=True) -> Any:
         Encode converts the state into a torch tensor so that Deepspeed serialization works.
         We don't encode any of the super() calls, but encode only the final dict.
         """
-        extra_state = super().get_extra_state(encode=not encode)
+        extra_state = super().get_extra_state(encode=False)
```

Comment:
why?

Reply:
Crashes when there are nested RenateBenchmarkingModules. The simplification is that all sub-modules are not encoded, and there is only one final encoding.